ansible/roles/common_installs/tasks/main.yml

```diff
@@ -33,6 +33,11 @@
   sudo: yes
   with_items:
     - less@1.3.1
+    - n@1.3.0
```

Comment:
Does this really install an executable called `n` on your path!? That's such a... presumptuous name for a utility for managing node's version...